src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -54,31 +55,9 @@ int nwgmax(const std::vector<int>& inteHead)
     return inteHead[19];
 }
 
-int groupType(const Opm::Schedule& sched,
```

Comment:
I'd really prefer if the `groupType()` helper function be kept.  Its implementation could be simplified in light of `Group2`'s abilities, but it is much easier to read
```C++
iGrp[ix] = groupType(group);
```
than the alternative inline expansion
```C++
if (group.wellgroup()) {
    iGrp[ix] = 0;
}
else {
    iGrp[ix] = 1;
}
```
The former is also more extensible if we ever implement slave and satellite groups.

Reply:
ok